README.md

```diff
@@ -142,7 +142,7 @@ The output from each stage of the pipeline feeds into the next, and you can thin
 
 By comparison, writing shell-like scripts in raw Go is much less convenient, because everything you do returns a different data type, and you must (or at least should) check errors following every operation.
 
-In scripts for system administration we often want to compose different operations like this in a quick and convenient way. If an error occurs somewhere along the pipeline, we would like to check this just once at the end, rather than after every operation.
+In scripts for system administration, we often want to compose different operations like this in a quick and convenient way. If an error occurs somewhere along the pipeline, we would like to check this just once at the end, rather than after every operation.
```

Comment:
Stat. The comma creates a breathing pause here which is not necessary for the rhythm of the sentence.